templates/tools/dockerfile/php7_deps.include

```diff
@@ -19,6 +19,7 @@ RUN apt-get update && apt-get install -y ${'\\'}
   re2c ${'\\'}
   time ${'\\'}
   unzip ${'\\'}
+  valgrind ${'\\'}
```

Comment:
I'm not convinced valgrind should be part of PHP's standard testing dependencies (it then bubbles up e.g. into interop test images, where it is not needed). How big is valgrind installation?

Reply:
@jtattermusch Thanks for the review.

So you are saying that, in addition to the `php` and `php7` base docker images we have now,  we should have a `php_memtest` base image, which should be completely separate from the rest, and only in there we install and run `valgrind`?

Reply:
adding a new image is an overkill. But I'd like to see if we could skip installing valgrind where it is not necessary (e.g. for interop tests) and I'd like to know how much it adds to the PHP size.
Also please note that linux docker images are not the only place where we run PHP tests (e.g. there are performance workers and tests on other platforms) where valgrind is not installed - so using valgrind should probably be optional.

Reply:
valgrind adds 0.35GB to the image size.

Reply:
So what we should do, is to add some logic to say, 1. if this is not interop test, 2. if it's linux only, install valgrind and run the memory leak test?

Reply:
Let's only add `<%include file="../../valgrind.include"/>` in the right docker images where they are useful. E.g. only add here after this line: https://github.com/grpc/grpc/blob/3278bdceda8030d5aa130f12765e5f07263c860d/templates/tools/dockerfile/test/php7_jessie_x64/Dockerfile.template#L21`

Reply:
@jtattermusch Thanks for helping. I just pushed changes into this PR followed by your comment and create a valgrind.include file for install valgrind command. But my question is what is the difference between adding include valgrind installation inside docker images and in php_deps.include ? Should <%include file="../../valgrind.include"/> will still always include valgrind installation?

Reply:
The difference is that like this valgrind won't be installed in the interop docker image (and interop docker images are being kept for backward compatibility suite).